packages/maskbook/src/plugins/BarnBridge/Import/BarnBridgeFrontEnd/web3/utils.ts

```diff
@@ -0,0 +1,143 @@
+import BigNumber from 'bignumber.js'
+
+export function getExponentValue(decimals = 0): BigNumber {
```

Comment:
plz merge the file non-adhoc function to `packages/web3-shared/src/utils/number.ts`